riff-raff/conf/evolutions/default/4.sql

```diff
@@ -0,0 +1,31 @@
+-- Read only user for sourcing service-catalogue data
```

Comment:
For some reason all the other evolutions were using `#` for comments, this is a little confusing as Postgres uses `--` for comments. I was getting a lot of syntax errors when writing my longer comments, so I've switched to using `--` which has fixed that.